workflows/VGP-assembly-v2/VGP-Bionano/Galaxy-Workflow-VGP_Bionano.ga

```diff
@@ -14,206 +14,199 @@
     ],
     "format-version": "0.1",
     "license": "CC-BY-4.0",
-    "release": "0.1",
```

Comment:
This commit deletes the "release" field for three out of four workflows. It needs to be restored.